Modules/privateEndpoint.bicep

```diff
@@ -15,8 +15,8 @@ param groupIds array = [
     'blob'
 ]
 
-var privateDnsZoneName = '${privateEndpointName}.blob.core.windows.net'
```

Comment:
For this line I would really like to keep privateEndpointName as a parameter to give more flexibility